test/evp_kdf_test.c

```diff
@@ -307,8 +311,25 @@ static int test_kdf_sshkdf(void)
     return ret;
 }
 
+static int test_kdf_get_kdf(void)
+{
+    const EVP_KDF *kdf1, *kdf2;
+    ASN1_OBJECT *obj;
+
+    return TEST_ptr(obj = OBJ_nid2obj(NID_id_pbkdf2))
+           && TEST_ptr(kdf1 = EVP_get_kdfbyname(LN_id_pbkdf2))
+           && TEST_ptr(kdf2 = EVP_get_kdfbyobj(obj))
+           && TEST_ptr_eq(kdf1, kdf2)
+           && TEST_ptr(kdf1 = EVP_get_kdfbyname(SN_tls1_prf))
+           && TEST_ptr(kdf2 = EVP_get_kdfbyname(LN_tls1_prf))
+           && TEST_ptr_eq(kdf1, kdf2)
+           && TEST_ptr(kdf2 = EVP_get_kdfbynid(NID_tls1_prf))
+           && TEST_ptr_eq(kdf1, kdf2);
```

Comment:
Code style nit: 4 space indent.

Reply:
(incidently, 4 space indent means that all the `TEST_` calls are niceiy aligned :wink:)

Reply:
now uses same spacing as provider tests.